extensions/kudo/client/src/main/java/io/fabric8/kudo/client/InstanceKudoClient.java

```diff
@@ -0,0 +1,284 @@
+package io.fabric8.kudo.client;
+
+import io.fabric8.kubernetes.api.model.ObjectMeta;
+import io.fabric8.kubernetes.api.model.ObjectMetaBuilder;
+import io.fabric8.kubernetes.api.model.ObjectReference;
+import io.fabric8.kubernetes.api.model.ObjectReferenceBuilder;
+import io.fabric8.kubernetes.client.Config;
+import io.fabric8.kubernetes.client.utils.Serialization;
+import io.fabric8.kudo.api.model.kubectl.*;
```

Comment:
What is this?
Does it contain kubernetes resources? If so does it have a version?

Reply:
```io.fabric8.kudo.api.model.kubectl.*``` it is in the implementation of Kudo. There is no version.